srcpkgs/emptty/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'emptty'
+pkgname=emptty
+version=0.1.0
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
+make_build_target=build
+make_install_target="install install-config install-runit"
```

Comment:
With the switch to `build_style=go`, you will either have to manually install everything except the executable (you can do this in `post_install()`) or you can invoke `make` in `post_install()` to install these components yourself.

I recommend manually installing these files instead of relying on the makefile, because there are only a handful of files and the following behavior changes from the makefile:
1. Manual pages should not be gzipped in Void;
2. When installing a runit service, it is preferred to create a symlink from `/etc/sv/<service>/supervise` to `/run/runit/supervise.<service>` so service state is not preserved in the file system. (This is handled automatically with the `vsv` helper function.)

Long term, you should upstream changes to the makefile that split out the man page and pam file from the `install` target, don't zip the man page, and add the proper `supervise` symlink to the `runit` service. Then, you can simplify your `post_install()` in an update to something like
```
make DESTDIR="${DESTDIR}" install-manual install-pam install-config install-runit
```